package.json

```diff
@@ -31,12 +34,14 @@
     "glob": "^7.0.3",
     "google-auto-auth": "^0.7.0",
     "got": "^5.7.1",
+    "husky": "^0.14.3",
```

Comment:
It probably installs the githooks as part of the install script

https://github.com/typicode/husky/blob/master/package.json#L11